zsh-functions/search.zsh

```diff
@@ -0,0 +1,3 @@
+function search() {
```

Comment:
Test?

Reply:
tests are for smuchks?

Reply:
this comment is gonna come back to haunt me